snmp/datadog_checks/snmp/snmp.py

```diff
@@ -500,6 +502,84 @@ def report_metrics(
                 metric_tags = tags + metric.tags
                 self.submit_metric(name, val, metric.forced_type, metric_tags, metric.options)
 
+    @staticmethod
+    def is_bandwidth_metric(name):
+        # type: (str) -> bool
+        return name in ['ifHCInOctets', 'ifHCOutOctets']
+
+    @staticmethod
+    def bandwidth_metric_name_to_bandwidth_usage_metric_name(name):
+        # type: (str) -> str
+        return 'ifBandwidthInUsage' if name == 'ifHCInOctets' else 'ifBandwidthOutUsage'
```

Comment:
(nit) I think it might be cleaner to use a mapping dict from metric name to bandwidth usage metric name instead of if/else conditions.

Reply:
A dict for two hardcoded values?... 🤔